src/seedu/addressbook/data/person/Person.java

```diff
@@ -21,8 +23,11 @@
     private Optional<Account> account = Optional.empty();
     private Fees fees;
     private final Set<Tag> tags = new HashSet<>();
+    //private final List<Assessment> assessments = new ArrayList<>();
     private Attendance attendance;
-
+    //private final Set<Tag> assessment = new HashSet<>();
+    //private Map<Assessment, Grades> assessment_dic = new HashMap<>();
```

Comment:
Remove unused codes if possible, it adds unnecessary clutter for everyone else.
I know it's for a future endeavor, but still.
TO BE CHANGED